pkg/terraform/install.go

```diff
@@ -57,10 +57,13 @@ func (m *Mixin) Install() error {
 			fmt.Fprintf(m.Err, "DEBUG: TF var file created:\n%s\n", string(vbs))
 		}
 	}
-	for _, k := range sortKeys(step.Vars) {
```

Comment:
We're losing this key sorting, not sure the benefit of it but might want to revert back to this, thoughts?

Reply:
can you use a modifier to sort and get the same behavior?  https://github.com/tidwall/gjson/blob/v1.14.2/SYNTAX.md#modifiers

Reply:
Yeah looking at that. Also might want to do the same thing with backend config